C/manual/tips-appendix.docbook

```diff
@@ -45,6416 +63,6935 @@
     </para>
 
     <sect2 id="fq-currencies">
-      <title>Finance::Quote Sources - Currency source</title>
+      <title>Finance::Quote Sources - Currency Source</title>
 
       <table frame="topbot" id="gnc-tbl-fq-currency-source">
         <title>Currency source for Finance::Quote</title>
 
-        <tgroup cols="3">
-          <thead>
+        <tgroup cols="3">
+          <thead>
+            <row>
+              <entry>
+                <para>&appname; Name
+                </para>
+              </entry>
+
+              <entry>
+                <para>Finance::Quote Name
+                </para>
+              </entry>
+
+              <entry>
+                <para>Notes
+                </para>
+              </entry>
+            </row>
+          </thead>
+
+          <tbody valign='middle'>
+            <row>
+              <entry>
+                <para>Currency
+                </para>
+              </entry>
+
+              <entry>
+                <para>currency
+                </para>
+              </entry>
+
+              <entry>
+                <para>End of 2017 the provider changed. Make sure, you updated F::Q to at least version 1.47 and follow
+                  the instructions at AlphaVantage below.
+                </para>
+              </entry>
+            </row>
+          </tbody>
+        </tgroup>
+      </table>
+    </sect2>
+
+    <sect2 id="fq-sources-single">
+      <title>Finance::Quote Sources - Individual Sources</title>
+      <sect3 id="fq-active-sources-single">
+        <title>Active Individual Quote Sources</title>
+
+        <table frame="topbot" id="gnc-tbl-fq-active-individual-source">
+          <title>Individual sources for quotes</title>
+
+          <tgroup cols="3">
+            <thead>
+              <row>
+                <entry>
+                  <para>&appname; Name
+                  </para>
+                </entry>
+
+                <entry>
+                  <para>Finance::Quote Name
+                  </para>
+                </entry>
+
+                <entry>
+                  <para>URL, Notes
+                  </para>
+                </entry>
+              </row>
+            </thead>
+
+            <tbody valign='middle'>
+              <row>
+                <entry>
+                  <para>AlphaVantage, US
+                  </para>
+                </entry>
+
+                <entry>
+                  <para>alphavantage
+                  </para>
+                </entry>
+
+                <entry>
+                  <note>
+                    <para>You need to</para>
+                    <procedure>
```

Comment:
At least Ubuntu seems to have problems. How about

1. Upgrade `<note>` to `<important>`,
2. Replace the introducting `<para>` by a `<title>`
`<procedure><title>API Key Required</title>`

Reply:
done